commcare_connect/templates/opportunity/user_profile.html

```diff
@@ -71,3 +81,48 @@ <h1 class="display-4 mb-0 text-center">{{access.display_name}}</h1>
     });
   </script>
 {% endblock %}
+
+{% block modal %}
+<div class="modal fade" id="suspendUserModal" tabindex="-1" aria-labelledby="suspendUserModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h1 class="modal-title fs-5" id="suspendUserModalLabel">Suspend User: {{ access.user.name }}</h1>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <form method="post" action="{% url "opportunity:suspend_user" org_slug=request.org opp_id=access.opportunity_id pk=access.pk %}">
+        <div class="modal-body">
+          {% csrf_token %}
+          <div class="mb-3">
+            <label for="reason" class="form-label">{% translate "Suspension Reason" %}</label>
+            <textarea class="form-control" id="reason" name="reason" rows="3"></textarea>
+          </div>
+          <table class="table">
+            <tbody>
+              <tr>
+                <th scope="row">{% translate "Deliveries waiting for Approval" %}</th>
+                <td>{{ pending_completed_work_count }}</td>
+              </tr>
+              <tr>
+                <th scope="row">{% translate "Payment Accrued" %}</th>
+                <td>{{ access.payment_accrued }}</td>
+              </tr>
+              <tr>
+                <th scope="row">{% translate "Payment Pending Disbursement" %}</th>
+                <td>{{ pending_payment }}</td>
+              </tr>
+            </tbody>
+          </table>
+          <div class="alert alert-danger" role="alert">
+            {% translate "You will be unable to approve payments or pay the user after suspension." %}
```

Comment:
This could be a bit clearer that you won't be able to approve payments for that user, rather than everyone. it is a little unclear now and could be scary.

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/319/commits/2946e237c6dea71eedbf5574e6d000ebdb7eef81